clients/bucket_schema/client.go

```diff
@@ -40,19 +40,20 @@ func (c Client) resolveMeasurement(ctx context.Context, ids orgBucketID, name st
 }
 
 func (c Client) resolveOrgBucketIds(ctx context.Context, params clients.OrgBucketParams) (*orgBucketID, error) {
-	if params.OrgID.Valid() && params.BucketID.Valid() {
-		return &orgBucketID{OrgID: params.OrgID.String(), BucketID: params.BucketID.String()}, nil
-	}
-
-	if params.BucketName == "" {
+	if params.BucketName == "" && !params.BucketID.Valid() {
 		return nil, errors.New("bucket missing: specify bucket ID or bucket name")
 	}
 
 	if !params.OrgID.Valid() && params.OrgName == "" && c.ActiveConfig.Org == "" {
 		return nil, errors.New("org missing: specify org ID or org name")
 	}
 
-	req := c.GetBuckets(ctx).Name(params.BucketName)
+	req := c.GetBuckets(ctx)
+	if params.BucketName != "" {
+		req = req.Name(params.BucketName)
+	} else {
+		req = req.Id(params.BucketID.String())
```

Comment:
Do we want to prioritize the ID over the name in the case when both are set? That'd be consistent with how we manage org name/ID pairs. Not sure if it matters in practice.

Reply:
I'll switch them around if it is consistent with other commands 👍🏻 

Thanks for the review!